controllers/managedocs_controller.go

```diff
@@ -95,6 +147,15 @@ func (r *ManagedOCSReconciler) reconcilePhases(req ctrl.Request) error {
 	}
 	r.managedOCS.Status.ReconcileStrategy = reconcileStrategy
 
+	configSecret := &corev1.Secret{}
+	err := r.Get(r.ctx, types.NamespacedName{Namespace: req.Namespace, Name: r.ConfigSecretName}, configSecret)
+	if err != nil {
```

Comment:
Two notes here:
1.  Can we merge these into the if? It's a go best practice 
```if err := r.Get(r.ctx, types.NamespacedName{Namespace: req.Namespace, Name: r.ConfigSecretName}, configSecret); err != nil {```

2. Can we move this inside the `setDesiredStorageCluster` func to prevent this logic (and reading of the secret) in case the reconcile strategy is `none`?

Reply:
This was done in this function in order to get the req.namespace to get the secret.
The other option is to pass the request as an argument to setDesiredStorageCluster.

Reply:
or same the namespace on the reconciler
```
r.namespace = req.namespace
```

Reply:
done